packages/base/src/3dview/mainview.tsx

```diff
@@ -103,7 +103,7 @@ export class MainView extends React.Component<IProps, IStates> {
     this._mainViewModel.renderSignal.connect(this._requestRender, this);
     this._mainViewModel.workerBusy.connect(this._workerBusyHandler, this);
 
-    this._raycaster.params.Line = { threshold: 50 };
+    this._raycaster.params.Line2 = { threshold: 50 };
```

Comment:
I am a bit confused by this.

- I see it working properly in Firefox, just like in your screencast. So far so good, but we're not using `Line2` in the codebase anymore, so why does it work?
- I do not see it working in Chromium for some reason.

So should we explicitly use `Line2` instead of `Line` for line meshes in the codebase so that it works in all cases? I would try that, but I really don't understand why it works in Firefox.